dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/StackJoinerManager.java

```diff
@@ -170,7 +170,7 @@ public void deactivate() {
     
     
     public synchronized void resolveStackJoinFeature(Map<String, Object> serverConfigMap) {
```

Comment:
Rename method name?

Reply:
The configuration value has changed, but the feature/functionality would still retain the stack joiner naming.

Reply:
That being said, besides the `LoggingConstants`  and `BaseTraceService` (in com.ibm.ws.logging) referring to it as `stackTraceSingleEntry` when referring to the explicitly to configuration value the `StackJoinerConfiguration` and `StackJoinerManager` should keep the stack joiner designation and include java doc on the functionality name and configuration value.